owlkettle/adw.nim

```diff
@@ -139,6 +139,35 @@ renderable Avatar of BaseWidget:
       size = 100
       showInitials = true
 
+renderable ButtonContent of BaseWidget:
+  label: string
+  iconName: string
+  useUnderline: bool ## Defines whether you can use `_` on part of the label to make the button accessible via hotkey. If you surround a part of the label text with `_` it will hide the `_` and activate the button if you press ALT + The first key of the surrounded label text. E.g. `_Button_ Text` will trigger the button when pressing `ALT + B`.
```

Comment:
Are you sure that the text needs to be surrounded by `_` rather than needing to insert a single `_` to mark the letter used for the shortcut?

Reply:
I am not, the GTK docs are pretty... meh to understand on that front, at least for me.

Just tried it with only one underscore and that also worked, so appears I was wrong on that one.